staging/src/k8s.io/apiserver/pkg/endpoints/handlers/patch.go

```diff
@@ -561,19 +347,6 @@ func (p *patcher) applyAdmission(ctx context.Context, patchedObject runtime.Obje
 	return patchedObject, p.admissionCheck(patchedObject, currentObject)
 }
 
-func (p *patcher) requestLoop(ctx context.Context) func() (runtime.Object, error) {
-	// return a function to catch ctx in a closure, until finishRequest
-	// starts handling the context.
-	return func() (runtime.Object, error) {
-		updateObject, _, updateErr := p.restPatcher.Update(ctx, p.name, p.updatedObjectInfo, p.createValidation, p.updateValidation)
-		for i := 0; i < MaxRetryWhenPatchConflicts && (errors.IsConflict(updateErr)); i++ {
```

Comment:
Update has an internal loop in addition to this, so doesn't this old code actually in the worst case try up to 25 times, not 5? I've been wondering this for a while now.

Reply:
> doesn't this old code actually in the worst case try up to 25 times, not 5?

I don't think so. The only internal loop I see is in GuaranteedUpdate, and only loops in response to optimistic lock conflicts when writing to etcd.

I only see two ways a conflict error would be returned from Update:
1. if the patch specified a stale RV
2. if the patch delta computation returned a conflict error

Neither of those cases would cause GuaranteedUpdate to loop.